packages/eslint-config-node/index.js

```diff
@@ -1,32 +1,53 @@
-module.exports = {
-  env: {
-    es2020: true,
-    node: true,
-    mocha: true,
-    commonjs: true,
-  },
-  extends: ["@toruslabs/eslint-config-typescript", "plugin:n/recommended", "plugin:security/recommended-legacy", "plugin:chai-expect/recommended"],
-  plugins: ["chai-expect", "security"],
-  parserOptions: {
-    ecmaVersion: 11,
-  },
-  rules: {
-    "import/extensions": [
-      "error",
-      "ignorePackages",
-      {
-        js: "never",
-        ts: "never",
-      },
-    ],
-    "n/no-unsupported-features/es-syntax": [
-      "error",
-      {
-        version: ">=16",
-        ignores: ["modules"],
+import chaiExpectPlugin from "eslint-plugin-chai-expect";
+import nodePlugin from "eslint-plugin-n";
+import pluginSecurity from "eslint-plugin-security";
+import torusTypescriptConfig from "@toruslabs/eslint-config-typescript";
+import globals from "globals";
+
+export default [
+  ...torusTypescriptConfig,
+  chaiExpectPlugin.configs["recommended-flat"],
+  nodePlugin.configs["flat/recommended-script"],
+  pluginSecurity.configs.recommended,
+  {
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        ...globals.mocha,
+        ...globals.commonjs,
+        ...globals.es2020,
       },
-    ],
-    "n/no-missing-import": 0,
-    "import/no-extraneous-dependencies": ["error", { devDependencies: ["**/*.test.ts", "**/*.test.tsx", "**/testHelpers.ts"] }],
+
+      ecmaVersion: 11,
```

Comment:
let's bump our code to support es2024. node 22 fully supports it.
exported code (library generated) should support node 20. (which is es2023)
so, eslint ecmaVersion should be 15 here

Reply:
we also can put es2024 here, should it still be 15 or es2024?